dev_guide/quota.adoc

```diff
@@ -243,3 +243,9 @@ The `*resource-quota-sync-period*` setting is designed to balance system
 performance. Reducing the sync period can result in a heavy load on
 the master.
 ====
+
+[[accounting_quota_dc]]
```

Comment:
changed to hyphens ... part of next push ... though I'm mention the other section heads appear to use underscores